### PR TITLE
[FLINK-23206] Minor tickets are moved to Not a Priority instead of cl…

### PR DESCRIPTION
**Pull request by knaufk**
…osing

### CODE REVIEW

**Comment by knaufk** [comment]
Once this is merged I will do a bulk change in Jira that
* moves all tickets resolution "Auto-Closed" to "Open" with priority "Not a Priority"
* replace label "auto-closed" with "auto-deprioritized-minor"

File: flink_jira_bot.py [hunk 1]
```diff
@@ -65,14 +64,16 @@ def get_args():
...
         jira, jira_bot_config["stale_assigned"], args.dryrun
     )
-    stale_minor_rule = StaleMinorRule(jira, jira_bot_config["stale_minor"], args.dryrun)
-    stale_major_rule = StaleMajorOrAboveRule(
+    stale_minor_rule = StaleUnassignedRule(
+        jira, jira_bot_config["stale_minor"], args.dryrun, "Major", "Not a Priority"
+    )
+    stale_major_rule = StaleUnassignedRule(
         jira, jira_bot_config["stale_major"], args.dryrun, "Major", "Minor"
     )
```
Comment (pnowojski) [inline flink_jira_bot.py]: How is it supposed to be working? We have now two `StaleUnassignedRule`s, both from `Major` one to `Minor` other to `Not a Priority` at the same time?

File: stale_unassigned_rule.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 import logging
 
 
-class StaleMajorOrAboveRule(FlinkJiraRule):
+class StaleUnassignedRule(FlinkJiraRule):
     """
     Tickets major and above without a fixVersion need an assignee, or an update within {stale_<blocker|critical|major>.stale_days},
```
Comment (pnowojski) [inline stale_unassigned_rule.py]: nit: outdated comment `Tickets major and above`.

**Review by pnowojski — COMMENTED** [review]
I might not be the best person to review this code (first time seeing it 😳 ) but I've left two comments.

File: stale_unassigned_rule.py (hunk 2 above)
Comment (knaufk) [reply stale_unassigned_rule.py]: Fixed.

File: flink_jira_bot.py (hunk 1 above)
Comment (knaufk) [reply flink_jira_bot.py]: It is not ;) One of the them should be Minor. Fixed.

**Comment by knaufk** [comment]
@pnowojski Thanks you for the review. Good to see that more people are familiarizing themselves with the code increasing the bus factor. I addressed your comments.

**Review by pnowojski — APPROVED** [review]
I'm not 100% sure but I think it LGTM :)

**Pull request merged**